packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -71,6 +71,7 @@ const enableKernelGC = true;
 //   $vatSlot is one of: o+$NN/o-$NN/p+$NN/p-$NN/d+$NN/d-$NN
 // v$NN.c.$vatSlot = $kernelSlot = ko$NN/kp$NN/kd$NN
 // v$NN.nextDeliveryNum = $NN
+// v$NN.t.startPosition = $NN
```

Comment:
`startPosition` is inclusive and `endPosition` is exclusive, right? So when they're equal, that means the transcript is empty? Might be worth a brief comment to remind us.

Reply:
Yes, that's right.  I will add a comment.